### PR TITLE
feat: tile links component

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: resources/assets/css/_components.css [hunk 1]
```diff
@@ -348,3 +353,22 @@
...
+.tile-link:hover {
+    @apply shadow-lg text-theme-primary-700;
+}
+
+.tile-links-show-more {
+    @apply tile-link font-semibold text-center rounded bg-white text-theme-primary-600 transition;
+}
+.tile-links-show-more:hover {
+    @apply bg-theme-primary-700 text-white;
+}
```
Comment (ItsANameToo) [inline resources/assets/css/_components.css:374]: msq

File: resources/views/tile-link.blade.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @endif
+>
+    <div class="flex flex-col justify-center items-center space-y-2 h-full font-semibold">
+        @if ($link['icon'] ?? false)
+            <x-ark-icon :name="$link['icon']" size="md" />
+        @endif
+
+        <div>{{ $link['title'] }}</div>
+    </div>
+</a>
```
Comment (ItsANameToo) [inline resources/views/tile-link.blade.php:32]: msq

File: resources/views/tile-links.blade.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            >
+                <div class="flex flex-col justify-center items-center space-y-2 h-full">
+                    <x-ark-icon name="hide" size="md" />
+
+                    <div>@lang('ui::general.hide')</div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (ItsANameToo) [inline resources/views/tile-links.blade.php:76]: msq

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Comment by alexbarnsley** [comment]
dun dun dunnn (removed from this PR - will do a separate one once the css here is merged)

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**